src/main/java/xfacthd/oretexgen/client/loader/OreMetadata.java

```diff
@@ -1,46 +1,82 @@
 package xfacthd.oretexgen.client.loader;
 
+import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
 import net.minecraft.resources.ResourceLocation;
 import net.minecraft.server.packs.metadata.MetadataSectionSerializer;
 import net.minecraft.server.packs.resources.Resource;
+import org.jetbrains.annotations.Nullable;
 import xfacthd.oretexgen.OreTextureGenerator;
 
 import java.io.IOException;
 
 public final class OreMetadata
 {
     private static final String DEFAULT_BACKGROUND = "minecraft:block/stone";
-    private static final OreMetadata DEFAULT = new OreMetadata(DEFAULT_BACKGROUND, false);
+    private static final OreMetadata DEFAULT = new OreMetadata(DEFAULT_BACKGROUND, ShadowMetadata.fromJson(new JsonObject()));
```

Comment:
Shouldn't this be `null` instead of `ShadowMetadata.fromJson(new JsonObject())` to match the old behaviour of the default meta as well as the behaviour of specified metadata without the `shadow` key since the shadow meta being non-null is used to decide that a shadow should be generated?

Reply:
Wait, yeah, the default should be null. The new one from the empty object is for the "default true" configuration